test/containers/ProjectPageSpec.js

```diff
@@ -1,40 +1,33 @@
-import setup from '../setup.js'
-setup()
-
 import test from 'tape'
 import React from 'react'
 import { MemoryRouter } from 'react-router-dom'
-import { createStore, applyMiddleware, compose } from 'redux'
 import { Provider } from 'react-redux'
-import thunk from 'redux-thunk'
-
-import rootReducer from '../../src/reducers'
+import { mount } from 'enzyme'
+import sinon from 'sinon'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'sinon'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->